### ISSUE TITLE
Confused by the config actor_rollout_ref.actor.use_kl_loss

### ISSUE THREAD

**Issue by hzy46** [volcengine/verl#750]
The logic seems wrong here:

 https://github.com/volcengine/verl/blob/main/verl/trainer/ppo/ray_trainer.py#L887-L893

The code logic is something like:
```python
if not self.config.actor_rollout_ref.actor["use_kl_loss"]:
    # apply kl penalty
else:
    # skip kl penalty
```

So, if actor_rollout_ref.actor.use_kl_loss = True, we don't apply kl penalty here.

If actor_rollout_ref.actor.use_kl_loss = False, we apply kl penalty.

This logic is confusing. Is it by-design or is the code wrong here?

**Comment by vermouth1992** [comment]
use_kl_loss means using kl in loss. So, if we use kl in loss, we are not applying it in the reward function. It's a little bit confusing and we will try to make it clear in the next release. Thanks.

**Comment by hzy46** [comment]
Maybe a comment can help address the confusion here. I will close this issue. Thanks.
